crates/transaction-emitter-lib/src/emitter/submission_worker.rs

```diff
@@ -34,7 +34,9 @@ use tokio::time::sleep;
 
 pub struct SubmissionWorker {
     pub(crate) accounts: Vec<Arc<LocalAccount>>,
-    client: RestClient,
+    clients: Arc<Vec<RestClient>>,
```

Comment:
nit: `Arc<[RestClient]>` will do?

Reply:
choose method above is a bit picky, doesn't work on &[RestClient] for some reason.